packages/@okta/vuepress-site/docs/guides/oauth-for-okta-apis/scopes/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Scopes & Supported Endpoints
+---
+Every action on an endpoint that supports OAuth 2.0 requires a specific scope. Okta scopes have the following format: `okta.<resource name>.<operation>`. For example, you can have resources that are users, clients, or apps with `read` or `manage` operations. The `read` scope is used to read information about a resource. The `manage` scope is used to create a new resource, manage a resource, or delete a resource. Use the `okta.<resource>.read` scopes to perform GET API operations and the `okta.<resource>.manage` scopes to perform POST, PUT, and DELETE API operations. The self scopes (`okta.<resource>.<operation>.self`> only allow access to the user who authorized the token. These scopes are used to perform end user API operations.
+
+The following table shows the scopes that are currently available as a part of Early Access (EA):
+
+| Scope                    | Description                                                            | API                                    |
+|--------------------------|------------------------------------------------------------------------|----------------------------------------|
+| `okta.apps.manage`       | Allows the app to create and manage Apps in your Okta organization     | [Apps API](/docs/reference/api/apps/)  |
+| `okta.apps.read`         | Allows the app to read information about Apps in your Okta organization| [Apps API](/docs/reference/api/apps/)  |
+| `okta.authorizationServers.manage`| Allows an app to manage authorization servers                 | [Authorization Servers API](/docs/reference/api/authorization-servers/)|
+| `okta.authorizationServers.read`| Allows an app to read authorization server information          | [Authorization Servers API](/docs/reference/api/authorization-servers/)|
+| `okta.clients.manage`    | Allows an app to manage all OAuth/OIDC clients and to create new clients| [Dynamic Client Registration API](/docs/reference/api/authorization-servers/)|
+| `okta.clients.read`      | Allows an app to read information for all OAuth/OIDC clients           | [Dynamic Client Registration API](/docs/reference/api/authorization-servers/)|
+| `okta.clients.register`  | Allows an app to create new OAuth/OIDC clients (but not read information about existing clients)| [Dynamic Client Registration API](/docs/reference/api/authorization-servers/)|
```

Comment:
these clients scopes link to the wrong documentation (should be /docs/reference/api/oauth-clients/)